inbox/events/google.py

```diff
@@ -488,6 +490,58 @@ def parse_calendar_response(calendar: Dict[str, Any]) -> Calendar:
     return Calendar(uid=uid, name=name, read_only=read_only, description=description)
 
 
+MAX_STRING_LENGTH = 8096
+MAX_LIST_LENGTH = 10
+STRING_VALIDATORS = [
+    attrs.validators.instance_of(str),
+    attrs.validators.max_len(MAX_STRING_LENGTH),
+]
+
+
+@attrs.frozen(kw_only=True)
+class EntryPoint:
+    uri: str = attrs.field(validator=STRING_VALIDATORS)  # type: ignore
```

Comment:
mypy barfs if you combine `str` validator with `max_len` but it works in practice.